tests/rustworkx_tests/digraph/test_clear.py

```diff
@@ -0,0 +1,66 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import rustworkx
+
+
+class TestClear(unittest.TestCase):
+    def test_clear(self):
+        dag = rustworkx.PyDAG()
+        node_a = dag.add_node("a")
+        node_b = dag.add_child(node_a, "b", {"a": 1})  # noqa: F841
+        node_c = dag.add_child(node_a, "c", {"a": 2})  # noqa: F841
```

Comment:
```suggestion
        dag.add_child(node_a, "b", {"a": 1})
        dag.add_child(node_a, "c", {"a": 2})
```

Reply:
Much better idea than what I had, thank you.